src/components/BackButton.tsx

```diff
@@ -0,0 +1,22 @@
+import './BackButton.scss'
+
+export type Props = {
+  disabled?: boolean
+}
+
+const BackButton = ({ disabled }: Props) => {
```

Comment:
```suggestion
import { FC } from 'react';

export interface BackButtonProps = {
  disabled?: boolean
}

const BackButton: FC<BackButtonProps> = ({ disabled }) => {
```

I have several suggestions here:

1. I'd make the props type name more specific to the component. Typically, I use the name of the component and add `Props` at the end
2. I'd make it an `interface` instead of a `type` since it's an object type. It's more suitable to use interfaces in this case
3. I'd recommend using the FC typing notation. I think it makes the code much cleaner